portality/app.py

```diff
@@ -443,6 +460,10 @@ def run_server(host=None, port=None, fake_https=False):
         that can help for debugging Plausible
     :return:
     """
+
+    if app.config.get('DEBUG_DEV_LOG', False):
+        setup_dev_log()
```

Comment:
I'm not totally clear what this does, do we not get DEBUG level logging just by being in DEBUG mode?

If this is relevant to all DEBUG level stuff, I'd suggest we just use this on DEBUG, but if we do need a new config value, can we also have it in `settings.py` set to `False` with a note about what it does?